sdk/cosmos/src/clients/cosmos_client.rs

```diff
@@ -19,6 +19,11 @@ use std::borrow::Cow;
 use std::fmt::Debug;
 use std::sync::Arc;
 
+/// The well-known account key used by Azure Cosmos DB Emulator.
+/// https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator?tabs=ssl-netstd21#connect-with-emulator-apis
```

Comment:
Though just a source comment, Azure SDKs asks that URLs not be pre-localized so that a browser's preferred locales are used.

```suggestion
/// https://docs.microsoft.com/azure/cosmos-db/local-emulator?tabs=ssl-netstd21#connect-with-emulator-apis
```

Reply:
Sounds good. There are lots of links that need to be updated, so I'll make another PR.

Reply:
https://github.com/Azure/azure-sdk-for-rust/pull/313